.coveralls.yml

```diff
@@ -0,0 +1,3 @@
+
+service_name: travis-pro
+repo_token: nUINoLdenxYURJlssXO3pDfWXAY1n6Xbp
```

Comment:
put the value of that token in `.env` file then you can call the value in this file like this `${process.env.REPO_TOKEN}`. Also remember to add an example in `.env.example` of every variable you put in `.env`